cc2538-bsl.py

```diff
@@ -1221,7 +1221,7 @@ def cli_setup():
                 raise Exception("NO CRC32 match: Local = 0x%x, "
                                 "Target = 0x%x" % (crc_local, crc_target))
 
-        if args.ieee_address != 0:
+        if args.ieee_address:
```

Comment:
```suggestion
        if args.ieee_address != 0 and args.ieee_address != None:
```

Reply:
Thanks for your review! How do you handle suggestions and review feedback in this project? May I commit and resolve this conversation, or is this up to someone else?

Reply:
Since im not part of the official "cc2538-bsl" Repo / Project Team i unfortunately can't answer your question.
For me it would not be a big deal if you would just commit it and resovle the conv.